addons/binding/org.openhab.binding.miele/ESH-INF/thing/dishwasher.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Dishwasher -->
+	<thing-type id="dishwasher">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="xgw3000" />
+		</supported-bridge-type-refs>
+
+		<label>Dishwasher</label>
+		<description>This is a Miele@Home compatible dishwasher</description>
+
+		<channels>
+			<channel id="state" typeId="state" />
+			<channel id="program" typeId="program" />
+			<channel id="phase" typeId="phase" />
+			<channel id="start" typeId="start" />
+			<channel id="duration" typeId="duration" />
+			<channel id="elapsed" typeId="elapsed" />
+			<channel id="finish" typeId="finish" />
+			<channel id="door" typeId="door" />
+			<channel id="switch" typeId="switch" />
```

Comment:
Wouldn't it be more expressive to call it `powerState` or something like that? Switch could be some switch on the device - not necessarily the power switch.